scipy/stats/_continuous_distns.py

```diff
@@ -3985,6 +3985,101 @@ def fit(self, data, *args, **kwds):
 laplace = laplace_gen(name='laplace')
 
 
+class laplace_asymmetric_gen(rv_continuous):
+    r"""An asymmetric Laplace continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `laplace_asymmetric` is
+
+    .. math::
+
+        f(x, \kappa) &= \frac{1}{\kappa+\kappa^{-1}}\exp(-x\kappa),\quad x\ge0\\
+                     &= \frac{1}{\kappa+\kappa^{-1}}\exp(x/\kappa),\quad x<0\\
+
+    for :math:`-\infty < x < \infty`, :math:`\kappa > 0`.
+
+    `laplace_asymmetric` takes ``kap`` as a shape parameter for
+    :math:`\kappa`. For :math:`\kappa = 1`, it is identical to a
+    Laplace distribution.
+
+    %(after_notes)s
+
+    See Also
+    --------
+    laplace : Laplace distribution
+
+    References
+    ----------
+    .. [1] Kozubowski TJ and Podg\'orski K. A Multivariate and
+           Asymmetric Generalization of Laplace Distribution,
+           Computational Statistics 15, 531--540 (2000).
+           :doi:`10.1007/PL00022717`
+
+    %(example)s
+
+    """
+    def _rvs(self, kap, size=None, random_state=None):
```

Comment:
No need to abbreviate `kappa` to `kap` here.  `kappa` is the name of parameter in the docstring and the description of the distribution, so it makes sense to keep it as `kappa` in the code.

Reply:
Done thanks.